src/org/simple/diary/DS_Impl.java

```diff
@@ -0,0 +1,41 @@
+package org.simple.diary;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DS_Impl implements DiaryService {
+    private final DR_Impl DiaryRepository = new DR_Impl();
+
+    /**
+     * 다이어리를 작성한다.
+     * @param writer 작성자
+     * @param title 다이어리 제목
+     * @param content 다이어리 내용
+     * @return 생성된 다이어리 객체
+     */
+    public Diary writeDiary(User writer, String title, String content){
+       Diary newDiary = new Diary();
+       newDiary.setUser(writer);
+       newDiary.setTitle(title);
+       newDiary.setContent(content);
```

Comment:
새로운 **Diary** 인스턴스가 생성되는 시점이므로, setter 보다는 생성자를 통해서 상태를 결정해주는 것이 좋아보입니다 :)
```suggestion
       Diary newDiary = new Diary(writer, title, content);
```